mlprimitives/jsons/mlprimitives.candidates.dsp.SpectralMask.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "mlprimitives.candidates.dsp.SpectralMask",
+    "author": "Rodrigo Diez <rjdiez@mit.edu>",

```

Comment:
Could you replace the "author" entry with a "contributors" list? We have recently changed this to allow having multiple contributors per primitive.

Reply:
ok, should I use "contributors" or "contributors list"?

Reply:
just "contributors", like this:
```
"contributors": ["Rodrigo Diez <rjdiez@mit.edu>"],
```

Reply:
ok